src/util/Angle.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Util class for angle related calculations.
+ */
+const AngleUtil = {
+  /**
+   * Determine the angle between two coordinates. The angle will be between
+   * -180° and 180°, with 0° being in the east. The angle will increase
+   * counter-clockwise.
+   *
+   * Inspired by http://stackoverflow.com/a/31136507
+   *
+   * @param {Array<Number>} start The start coordinates of the line with the
+   *   x-coordinate being at index `0` and y-coordinate being at index `1`.
+   * @param {Array<Number>} end The end coordinates of the line with the
+   *   x-coordinate being at index `0` and y-coordinate being at index `1`.
+   * @returns {Number} the angle in degrees, ranging from -180° to 180°.
+   */
+  calcAngle (start, end) {
+    const dx = start[0] - end[0];
+    const dy = start[1] - end[1];
+    // range (-PI, PI]
+    let theta = Math.atan2(dy, dx);
+    // rads to degs, range (-180, 180]
+    theta *= 180 / Math.PI;
+    return theta;
+  },
+
+  /**
+   * Determine the angle between two coordinates. The angle will be between
+   * 0° and 360°, with 0° being in the east. The angle will increase
+   * counter-clockwise.
+   *
+   * Inspired by http://stackoverflow.com/a/31136507
+   *
+   * @param {Array<Number>} start The start coordinates of the line with the
+   *   x-coordinate being at index `0` and y-coordinate being at index `1`.
+   * @param {Array<Number>} end The end coordinates of the line with the
```

Comment:
maybe here as type an OpenLayers coordinate can be used: https://openlayers.org/en/latest/apidoc/module-ol_coordinate.html#~Coordinate